pipeline/unified/channel/sign-release-package-windows.yaml

```
@@ -23,6 +23,45 @@ jobs:
                 artifact: ${{ parameters.unsignedArtifactName }}
                 path: '$(System.DefaultWorkingDirectory)/drop/electron/unified-${{ parameters.channel }}'
 
+          - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+            displayName: 'Sign artifact binaries'
+            inputs:
+                ConnectedServiceName: 'ESRP Code Signing'
+                FolderPath: '$(System.DefaultWorkingDirectory)/drop/electron/unified-${{ parameters.channel }}'
+                Pattern: '*.exe, *.dll'
+                signConfigType: inlineSignParams
+                inlineOperation: |
+                    [
+                        {
+                        "keyCode": "CP-230012",
```

For 3rd party cert, this will be:
```suggestion
                        "keyCode": "CP-231522",
```